recipe/meta.yaml

```diff
@@ -72,11 +93,11 @@ outputs:
         - wheel
       run:
         - python
-        - numpy <1.29,>=1.21
-        - pandas <2.2.0,>=2.0.0
-        - boto3 <2,>=1.10
-        - psutil <6,>=5.7.3
-        - tqdm <5,>=4.38
+        - numpy {{ numpy }}
+        - pandas {{ pandas }}
```

Comment:
It does have the benefit of ensuring that all of the repeated pinnings get updated together. Could we consider eliminating duplicate pinnings where they'll be inherited from a subpackage? For example, we can remove `numpy`, `pandas`, `boto3`, `psutil` and `tqdm` from all outputs besides `autogluon.core`

Reply:
I renamed `numpy` and `pandas` with the suffix `_local` to avoid confusion with CBC parameters. I also removed the duplicate dependencies as suggested by @ViridianMelody.